### PR TITLE
translate weapon facts.

### PR DESCRIPTION
**Pull request by Deerlord0**

### CODE REVIEW

File: resource/reactivedrop_japanese.txt [hunk 1]
```diff
@@ -5877,22 +5877,22 @@
 "[english]rd_weapon_fact_projectile_speed"		"Projectile Speed"
 "rd_weapon_fact_projectile_speed_before"		""
 "[english]rd_weapon_fact_projectile_speed_before"		""
-		"rd_weapon_fact_projectile_speed_after"		" per second"
-		"[english]rd_weapon_fact_projectile_speed_after"		" per second"
-		"rd_weapon_fact_piercing"		"Piercing Chance"
-		"[english]rd_weapon_fact_piercing"		"Piercing Chance"
+"rd_weapon_fact_projectile_speed_after"		" 毎秒"
```
Comment (BenLubar) [inline resource/reactivedrop_japanese.txt:5880]: Is `1000 ユニット 毎秒` correct? Google Translate puts the words in a different order.

Comment (BenLubar) [reply resource/reactivedrop_japanese.txt:5880]: Google Translate says `毎秒1000単位`, which looks like the translation for "per second" should go in `rd_weapon_fact_projectile_speed_before` and this one should be blank.

(But I don't know Japanese, so ignore me if I'm wrong.)

Comment (Deerlord0) [reply resource/reactivedrop_japanese.txt:5880]: I think "units" means 1.75 inch, right? Then "ユニット" is good, "単位" is wrong.
Like "feet" is "フィート", not "足" when use as length.
rarely or almost never use "単位" as distance or length. uncommonly use as quantity of uncountable things like liquid.

As projectile speed
"秒速 1000 ユニット" (this is best)
"毎秒 1000 ユニット" (looks a bit weird)
"1000 ユニット/秒" (this is possible choice if those words will come after the number) 
would be good.

Gonna change
"rd_weapon_fact_projectile_speed_after"		" 毎秒"
to
"rd_weapon_fact_projectile_speed_after"		" /秒" (or '/秒 for escape?)
then it looks better for Japanese people.

Comment (BenLubar) [reply resource/reactivedrop_japanese.txt:5880]: In that case,
`"rd_weapon_fact_projectile_speed_before" "秒速 "`

and make `"rd_weapon_projectile_speed_after" ""`

Comment (Deerlord0) [reply resource/reactivedrop_japanese.txt:5880]: Ah, now I see those _before and _after means and can imagine how they will be shown in game. thanks.

**Pull request merged**